presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DbSourceExactMatchSelector.java

```diff
@@ -63,7 +66,11 @@ public DbSourceExactMatchSelector(String environment, ResourceGroupsDao dao)
             }
 
             try {
-                return Optional.of(new SelectionContext<>(resourceGroupIdCodec.fromJson(resourceGroupId), emptyVariableMap()));
+                ResourceGroupId resourceGroupId1 = resourceGroupIdCodec.fromJson(resourceGroupId);
```

Comment:
rename this to `resourceGroupId` and the one on line 54 to `resourceGroupIdString`